docu/Concepts/Snippets/Iota/ColoredCoins.md

```diff
@@ -0,0 +1,34 @@
+# Wie Colored Coins in Iota funktionieren
+## Schöpfung
+- Colored Coins werden bei Iota mit einer speziellen Transaktion erzeugt
+- Die Farbe des neuen Coins wird durch den Transaktionshash beschrieben
+- Die einmal erzeugte Menge von Colored Coins ist fest
```

Comment:
Und genau hier sage ich: Veto!
Wir haben eine variierende Geldmenge. Die vorgeschlagene Technologie bildet aber eine feste Geldmenge ab. Ich bin ziemlich sicher, dass man da rumtricksen kann, um das trotzdem abzubilden. Ich krieg auch ein Nagel mit nem Schraubenzieher in die Wand.

Reply:
Soweit ich es verstanden habe, kann man die geschöpfte Menge an Colored IOTAs nicht nachträglich erhöhen. Man müsste eine übergroße Menge an IOTAs schöpfen, die dann auf einem Sperrkonto halten und nur für die Gradido-Schöpfung freigeben. Wenn das Sperrkonto korrumpiert wird, wird das Netz mit falschen Gradidos geflutet.

Reply:
Auf einer Übersichtsseite von Iota steht, das man wohl neue Colored Coins schaffen muss wenn man mehr braucht und dann z.B. mit einem SmartContract die alten durch die neuen austauschen muss/kann.